package.json

```diff
@@ -39,5 +39,9 @@
     "node-abi": "^3.34.0",
     "cpu-features": "./.yarn/packages/empty-package",
     "electron-builder": "23.0.2"
+  },
+  "volta": {
+    "node": "16.19.1",
+    "yarn": "1.22.21"
```

Comment:
Is this intended? It's all good if it is. Maybe I'm missing something.

Reply:
Didn't actually mean to check this in. It doesn't make a difference for building or anything, it's just for local machines if you happen to use Volta

Reply:
Oh ok. That's cool.